docs/concepts/work-queues.md

```diff
@@ -12,40 +12,103 @@ tags:
     - concurrency limits
 ---
 
-# Work Queues and Agents
+# Agents and Work Queues
```

Comment:
Looks like the title is set in mkdocs.yaml and needs to be changed there so "Agents and Work Queues" appears in the nav

Reply:
Thanks!